packages/apps/src/Endpoints/Network.tsx

```diff
@@ -25,7 +25,11 @@ function NetworkDisplay ({ apiUrl, className = '', setApiUrl, value: { icon, isC
   );
 
   const _selectUrl = useCallback(
-    () => setApiUrl(name, providers[Math.floor(Math.random() * providers.length)].url),
+    () => {
+      const filteredProviders = providers.filter((p) => !p.name.startsWith('light client'));
```

Comment:
Not complaining, but we should pass through the `isLightClient` thing here. Can be done later... but probably jut need a comment to make sure we pass that through the chain somehow.

Reply:
> Not complaining, but we should pass through the `isLightClient` thing here. Can be done later... but probably jut need a comment to make sure we pass that through the chain somehow.

You want to add it in this PR or open an issue for later on?

Reply:
We can just open an issue.

Reply:
Opened [this issue](https://github.com/polkadot-js/apps/issues/5734).

Is there anything else pending for merging this?